tools/cldr-code/src/main/java/org/unicode/cldr/util/DiffLanguageGroups.java

```diff
@@ -1,14 +1,18 @@
 package org.unicode.cldr.util;
```

Comment:
```suggestion
package org.unicode.cldr.tool;
```

If this is really a tool, it should go under the tool package.

Reply:
Agreed. I just didn't want to do that in a single PR.